src/ECS/Systems/Systems.cpp

```diff
@@ -261,6 +264,15 @@ namespace Systems {
         }
     }
 
+    static void sendEnemyDeath(std::size_t id)
```

Comment:
warning here because send enemy death from client is not already implemented, it lead to an unused value ('id') only on client compilation